### DISCUSSION TITLE
How to setup grpc with VRTS?(help)

### DISCUSSION THREAD

**Discussion by markusmat97** [ni/grpc-device#806] [Q&A]
Hi,
I am trying to setup grpc with the VRTS hardware.The VRTS hardware has a PXIe-8881 Embedded controller,PXIe 5551 adn PXIe-5842.
i want to remotely control the VRTS with gRPC.Is there any other ways of controlling VRTS other than remote desktop connection?
i am new to gRPC setup,any help will be greatful.

**Comment by DavidCurtiss** [comment]
It depends on what pieces you need to control remotely. The VRTS [Getting Started guide](https://www.ni.com/docs/en-US/bundle/vrts-with-1-ghz-bandwidth-getting-started/page/installing-sw.html) says to install NI-RFSA, NI-RFSG, NI-RFmx SpecAn, and NI-RFmx DeMod. As you can see in our [Supported Drivers](https://github.com/ni/grpc-device#supported-ni-drivers) list for gRPC, we support all of those except for NI-RFmx DeMod.

Also note that we currently only provide these low-level driver accessors over gRPC. You wouldn't be able to use it with the NI-VRTS instrument driver, for example.

If you'd like to continue investigating this possibility, though, I'd suggest having a look at the corresponding examples, like [these](https://github.com/ni/grpc-device/tree/main/examples/nirfmxspecan) for NI-RFmx SpecAn, to see if you can find something that might help fulfill your requirements.
